Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -188,6 +265,22 @@ public class KotlinTokenizer: SwiftTokenizer {
             node)]
     }
 
+    private func indexOfBalancingBracket(_ tokens:[Token]) -> Int? {
+        var count = 0
+        for (i, t) in tokens.enumerated() {
+            if t.kind == .startOfScope && t.value == "(" {
+                count += 1
+            }
+            if t.kind == .endOfScope && t.value == ")" {
+                count -= 1
+                if count <= 0 {
```

Comment:
Maybe better `== 0` so in case you pass tokens that closes a bracket before opening then it returns nil?